TRANSPARENCY_FAQ.md

```diff
@@ -44,7 +44,10 @@ Microsoft Agent Framework relies on existing LLMs. Using the framework retains c
 
 - **Platform Requirements**: Python 3.10+ required, specific .NET versions (.NET 8.0, 9.0, netstandard2.0, net472)
 - **API Dependencies**: Requires proper configuration of LLM provider keys and endpoints
-- **Orchestration Features**: Advanced orchestration patterns like GroupChat, Sequential, and Concurrent orchestrations are "coming soon" for Python implementation
+- **Orchestration Features**: Advanced orchestration patterns including GroupChat, Sequential, and Concurrent workflows are available today. See:
```

Comment:
Do we need specific sample links in the higher-level TRANSPARENCY_FAQ doc? I'd agree that they should be present in the Python README, but this is supposed to pertain to any MAF language - not strictly Python.